src/polybench/POLYBENCH_ATAX-Cuda.cpp

```diff
@@ -109,41 +112,74 @@ void POLYBENCH_ATAX::runCudaVariant(VariantID vid)
 
     POLYBENCH_ATAX_VIEWS_RAJA;
 
-    using EXEC_POL =
+    using EXEC_POL1 =
       RAJA::KernelPolicy<
         RAJA::statement::CudaKernelAsync<
-          RAJA::statement::For<0, RAJA::cuda_threadblock_exec<block_size>,
-            RAJA::statement::Lambda<0>,
-            RAJA::statement::For<1, RAJA::seq_exec,
-              RAJA::statement::Lambda<1>
+          RAJA::statement::Tile<0, RAJA::statement::tile_fixed<block_size>, RAJA::cuda_block_x_loop,
+            RAJA::statement::For<0, RAJA::cuda_thread_x_loop,
```

Comment:
We should consistently use cuda_thread_x_direct in fixed tiles.

Reply:
Yes, thanks! I will commit changes shortly.